.github/workflows/spellcheck.yml

```diff
@@ -0,0 +1,20 @@
+name: Spellcheck
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  spellcheck:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "16"
+          cache: "npm"
+      - run: npm ci
+      - name: Run spellcheck
+        run: npx cspell lint projects/**/*.{ts,md,js} --show-suggestions
```

Comment:
<details>
<summary>There are a ton of false positives getting logged on code files. Examples inside.</summary>

```plaintext
 13/221 ./projects/arrays/text-processor/src/index.test.ts 458.14ms X
/home/runner/work/projects/projects/projects/arrays/text-processor/src/index.test.ts:66:17 - Unknown word (abcdefghi) Suggestions: []
/home/runner/work/projects/projects/projects/arrays/text-processor/src/index.test.ts:66:61 - Unknown word (abcdefghi) Suggestions: []
```

```plaintext
 33/221 ./projects/configuration-options/secret-messages/01-encoded-outputs/index.verify.ts 316.61ms X
/home/runner/work/projects/projects/projects/configuration-options/secret-messages/01-encoded-outputs/index.verify.ts:12:14 - Unknown word (mkstx) Suggestions: [mktx, MKTX, masts, misti, mists]
/home/runner/work/projects/projects/projects/configuration-options/secret-messages/01-encoded-outputs/index.verify.ts:13:15 - Unknown word (uytm) Suggestions: [yum, ubyte, ulta, unto, upto]
```

```plaintext
/home/runner/work/projects/projects/projects/type-operations/template-literal-type-shenanigans/03-split-on/index.test.ts:7:46 - Unknown word (bbaabb) Suggestions: [babas, baba, baobab, blabby, baa's]
/home/runner/work/projects/projects/projects/type-operations/template-literal-type-shenanigans/03-split-on/index.test.ts:8:42 - Unknown word (bbaabb) Suggestions: [babas, baba, baobab, blabby, baa's]
/home/runner/work/projects/projects/projects/type-operations/template-literal-type-shenanigans/03-split-on/index.test.ts:9:48 - Unknown word (ababa) Suggestions: [abba, abaca, aqaba, Abba, Aqaba]
```
</details>

Do you see a lot of benefit in running on `.ts` and `.js` files?